torch/ao/quantization/experimental/quantizer.py

```diff
@@ -33,10 +33,13 @@ def quantize(self, tensor2quantize: Tensor):
         result = torch.tensor([])
 
         # map float_to_apot over tensor2quantize elements
-        tensor2quantize = tensor2quantize.apply_(lambda x: float_to_apot(x,
-                                                                         self.quantization_levels,
-                                                                         self.level_indices,
-                                                                         self.alpha))
+        tensor2quantize = tensor2quantize.detach().apply_(lambda x: float_to_apot(x,
```

Comment:
have the lambda function issues with jit been resolved?

Reply:
yep! the jit issues were only caused by the helper function that prints the size of the model, but i can use a different function `state_dict` to get the size of the model